xml/coast_of_bays_cioos.xml

```diff
@@ -58,6 +59,49 @@
       <mri:abstract>
         <gco:CharacterString>Upon a recent rapid increase of the finfish aquaculture industry in the Coast of Bays, an area of the South Coast of Newfoundland (9 fold production growth from 2003 to 2013), Fisheries and Oceans Canada (DFO) carried out a research project to better understand the physical oceanography of the area. This report is the second of a series aiming to provide an oceanographic knowledge baseline of the Coast of Bays (i.e., data and analyses) to help manage and ensure the sustainable growth of the aquaculture industry. This report presents the analysis of 790 water profiles of temperature, salinity and dissolved oxygen concentration sampled at 276 stations over the course of a 5-year oceanographic program (2009 13). The profiles were analyzed using along-channel vertical sections plots (i.e., transects), surface property maps and water masses statistics.</gco:CharacterString>
       </mri:abstract>
+      <mri:resourceConstraints>
+          <mco:MD_Constraints>
+            <mco:useLimitation xsi:type="lan:PT_FreeText_PropertyType">
+              <gco:CharacterString>Open Government Licence - Canada (http://open.canada.ca/en/open-government-licence-canada)</gco:CharacterString>
+              <lan:PT_FreeText>
+                <lan:textGroup>
+                  <lan:LocalisedCharacterString locale="#fra">Licence du gouvernement ouvert - Canada (http://ouvert.canada.ca/fr/licence-du-gouvernement-ouvert-canada)</lan:LocalisedCharacterString>
+                </lan:textGroup>
+              </lan:PT_FreeText>
+            </mco:useLimitation>
+            <mri:reference>
+              <cit:CI_Citation>
+                <cit:title>
+                  <gco:CharacterString>Coast of Bays seawater vertical and horizontal structure: Hydrographic structure, spatial variability and seasonality, 2009-2013</gco:CharacterString>                </cit:title>
+                <!-- mandatory (dataType= creation) by constraint enforced by Schematron rule -->
+                <cit:onlineResource>
+                  <cit:CI_OnlineResource>
+                    <cit:linkage>
+                            <gco:CharacterString>https://open.canada.ca/data/en/dataset/a650f132-7502-402f-a42e-7ba4dea3f829</gco:CharacterString>

```

Comment:
Indenting for this line looks off - you may want to set your tabs to two spaces for XML edits if you haven't already?

Reply:
Good eye! I didn't notice that, I updated PyCharm settings for the XML editor to use 2 spacing for both tab size and indents.